src/client/theme-default/components/VPDocAsideOutline.vue

```diff
@@ -45,16 +40,15 @@ function handleClick({ target: el }: Event) {
 
         <ul class="root">
           <li
-            v-for="{ text, link, children, hidden } in resolvedHeaders"
-            v-show="!hidden"
```

Comment:
Currently the `hidden` property is totally redundant because it would always be `undefined`, so I removed it for simplicity